

public class ExceptionTest1 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		int n=5;
		int[]  x;
		try {
            x=new int[n];   
            System.out.println("°}¦C«Å§i§¹¦¨");
		}catch(NegativeArraySizeException  e)
		{
			System.out.println(e);
		}
		System.out.println("end");
		
	}

}
